packages/block-editor/src/components/block-list/index.js

```diff
@@ -6,13 +6,8 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { Component, createRef } from '@wordpress/element';
-import {
-	withSelect,
-	withDispatch,
-	AsyncModeProvider,
-} from '@wordpress/data';
-import { compose } from '@wordpress/compose';
+import { useEffect, useRef } from '@wordpress/element';
+import { AsyncModeProvider, useSelect, useDispatch } from '@wordpress/data';
```

Comment:
Very minor nitpick: alphabetically, shouldn't `useDispatch` come before `useSelect`?

Reply:
Ah, I never cared to use any specific order. If we want these ordered we should add it to the linter.

Reply:
Yeah, good point. Maybe I'll make an issue for that later.